web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx

```diff
@@ -0,0 +1,3 @@
+export function SendCoins() {
+  return <h1>Send coins </h1>;
+}
```

Comment:
```suggestion
function SendCoins() {
  return <h1>Send coins </h1>;
}

export default SendCoins;
```

Reply:
is it best practice ?

Reply:
Yes! For pages is better to have this format. But if it's a component, we have `import { MyComponent } from "......";`.

Reply:
What did you think @mazmassa ?

Reply:
Let me just understand first why are we exporting one <h1> code!? 🤔

Reply:
it's just a placeholder :)

Reply:
What you mean is that will become a more complex component or a page!?

Reply:
This will be a component !

Reply:
Ok! Got it!